docs/index.html

```diff
@@ -0,0 +1,30 @@
+<html>
+  <head>
+    <script src="Chart.bundle.js"></script>
+    <script src="moment.js"></script>
+    <script src="Chart.Financial.js" type="text/javascript"></script>
+		<link rel="stylesheet" type="text/css" href="style.css">
+    <title>Financial | Chart.js</title>
+  </head>
+  <body>
+  	<h1>Chart.js - Financial chart</h1>
+  	<p>Source code for these examples is available here : <a href="https://github.com/loicbourgois/chartjs-chart-financial/tree/master/docs">chartjs-chart-financial/docs</a>.</p>
```

Comment:
this should probably say chartjs instead of loicbourgois?

Reply:
fixed